runtime/syntax/kvlang.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
What it's about that `---` in the first line? :smile:

Reply:
The `---` in yaml represents the beginning of an object, I put it so yamllint doesn't complain.

I can remove it if there is a problem

Reply:
Never to old to learn...
I'm currently puzzled if it's then better to add them in one bunch to all of the YAMLs. Currently I'm missing the analogy to the present definitions, especially since the `---` aren't mandatory. I'm torn. :thinking:

Reply:
It's yaml so the default is that it doesn't have a default, it's not mandatory and it won't change the behavior of the syntax, I don't know if it's worth the effort. I believe that the ideal would be to remove these `---` to keep the others as they are